ocaml/xapi/xapi_host.ml

```diff
@@ -792,18 +792,12 @@ let management_reconfigure ~__context ~pif =
   if Db.Pool.get_ha_enabled ~__context ~self:pool then
     raise (Api_errors.Server_error(Api_errors.ha_is_enabled, []));
 
-  (* Plugging a bond slave is not allowed *)
-  if Db.PIF.get_bond_slave_of ~__context ~self:pif <> Ref.null then
```

Comment:
That's good!
Except (my fault) there's no need for the `for_mgmt` parameter: it's misleading, since we always want to use the pif for management.
(I don't know what I was thinking when I suggested that parameter on Friday... somehow I had ended up halfway to considering a more general function checking if a pif can be plugged.)